modules/animationqt/CMakeLists.txt

```diff
@@ -38,6 +38,9 @@ qt5_add_resources(QRC_FILE "${CMAKE_CURRENT_SOURCE_DIR}/resources/animation.qrc"
 
 #--------------------------------------------------------------------
 # Create module
-ivw_create_module(${SOURCE_FILES} ${HEADER_FILES} ${QRC_FILE})
+ivw_create_module(NO_PCH ${SOURCE_FILES} ${HEADER_FILES} ${QRC_FILE})
```

Comment:
what does the NO_PCH means? Does it mean we tell the ivw_create_module function to not set PCH, but we still we have it since we enable it a few lines bellow this?

Reply:
it means that cotire is not run immediately when creating the module. One has to run cotire explicitly afterwards. This was necessary to be able to exclude certain paths.